core/src/main/java/org/testcontainers/utility/CommandLine.java

```diff
@@ -57,15 +59,19 @@ public static String runShellCommand(String... command) {
      */
     public static boolean executableExists(String executable) {
 
+        String winExecutable = executable + ".exe";
+
         // First check if we've been given the full path already
         File directFile = new File(executable);
-        if (directFile.exists() && directFile.canExecute()) {
+        File windowsExeFile = new File(winExecutable);
```

Comment:
I've just remembered that on Windows we can directly run not only ".exe" files but also ".bat", ".cmd" and (if PowerShell is a default shell) ".ps1". 
It is senseless in context of test-containers (as we know that "docker-compose.exe" is ".exe" and I doubt that this ever changes) but if we count CommandLine.java as a general purpose utility class it seems that we should cover these cases too (check windows executable extensions in a loop).
What do you think about ?